src/scripts/main.js

```diff
@@ -18,10 +18,26 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+let tree = document.querySelector('#tree');
+
+tree = document.createElement('div');
+tree.id = 'tree';
+document.body.appendChild(tree);
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+    ul.appendChild(li);
+  }
+  element.appendChild(ul);
 }
```

Comment:
The `createTree` function correctly creates a nested list from the provided data, but it does not return anything. This means that while the list is created and appended to the provided element, it cannot be used or manipulated after the function call. Consider returning the created list from this function.